k8s/models/policy_v1_pod_disruption_budget.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+# -*- coding: utf-8
+
+# Copyright 2017-2019 The FIAAS Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from .common import ObjectMeta, LabelSelector, Condititon
+from ..base import Model
+from ..fields import Field, ListField
+
+
+class PodDisruptionBudgetSpec(Model):
+    minAvailable = Field(str, alt_type=int)
+    maxUnavailable = Field(str, alt_type=int)
+    selector = Field(LabelSelector)
+    unhealthyPodEvictionPolicy = Field(str)
+
+
+class PodDisruptionBudgetStatus(Model):
+    condition = ListField(Condititon)
```

Comment:
"condition" could be changed to plural "conditions" if you feel like it.

And let me know if you're neutral/like/dislike this kind of detailed comments.

Reply:
Thanks, I very much appreciate it

Reply:
The field is named conditions (plural) in the API, so I think the field here has to match it

Reply:
Agree with oyvind:
https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/V1PodDisruptionBudgetStatus.md#properties

Reply:
👍 will fix